conans/client/package_layouts/package_user_layout.py

```diff
@@ -0,0 +1,17 @@
+# coding=utf-8
+
+from conans.util.files import load
+from conans.client.package_layouts.package_base_layout import PackageBaseLayout
+
+
+LINKED_FOLDER_SENTINEL = '.linked_package'
+
+
+class PackageUserLayout(PackageBaseLayout):
```

Comment:
If not used, I would skip this class by now

Reply:
Let's keep it here until the PR from `feature/editable-packages` to `develop`. Let's see how much it is used in the final implementation.